### ISSUE TITLE
Some programs don't appear

### ISSUE THREAD

**Issue by Foggalong** [ozonos/atom-panel#8]
If I'm understanding rightly it's supposed to take all the stuff from notification panel and put it in the top bar? Currently neither Rhythmbox nor Transmission will go up there. They seem to pick one of the two (usually notification pannel) at random :(

**Comment by me4oslav** [comment]
The desired behaviour is that only legacy tray - chrome, transmission, vlc, dropbox ... go there. Is that what happens no?

**Comment by cldx** [comment]
This is a general issue that is tied to the current development of Gnome, not by causing it, but support for Legacy Icons will be redone in Shell in the near future just like the notifications just got redesigned, so we should have an eye on what they are doing to figure it out and react accordingly. The best right now would probably be to bug some Gnome Developers about where they are headed with the whole "We want Tray Icons/We hate Tray Icons" debacle.

**Comment by Foggalong** [comment]
@me4oslav no, it's hit and miss whether they appear or not which will certainly be confusing for new users :/

**Comment by me4oslav** [comment]
:(
